src/components/DigitizeProjectInstructions.vue

```diff
@@ -7,126 +7,85 @@ export default defineComponent({
-            </v-col>
-            <v-col>{{ $t('digitizeProjectInstructions.drawNewFeature') }}</v-col>
-          </v-row>
-          <v-row align="center" v-if="drawType != 'Box'" dense>
-            <v-col cols="auto" class="mr-4">
-              <v-btn icon="mdi-vector-polyline-edit" color="primary" variant="tonal" rounded="0" />
-            </v-col>
-            <v-col>{{ $t('digitizeProjectInstructions.modifyFeature') }}</v-col>
-          </v-row>
-          <v-row align="center" dense>
-            <v-col cols="auto" class="mr-4">
-              <v-btn icon="mdi-delete" color="warning" variant="tonal" rounded="0" />
-            </v-col>
-            <v-col>{{ $t('digitizeProjectInstructions.deleteFeature') }}</v-col>
-          </v-row>
-        </div>
-        <div class="text-h6 mt-10">{{ $t('findProjectInstructions.useButtonsToNavigate') }}</div>
-        <div class="text-p mt-2">
-          <v-row class="align-center" dense>
-            <v-col cols="auto" class="mr-4">
-              <v-btn icon="mdi-chevron-left" color="secondary" class="mr-2" variant="text" />
-              <v-btn icon="mdi-chevron-right" color="secondary" variant="text" />
-            </v-col>
-            <v-col>{{ $t('digitizeProjectInstructions.move') }}</v-col>
-          </v-row>
-          <div class="text-h6 mt-10">{{ $t('findProjectInstructions.toggleOpacityTitle') }}</div>
-          <div class="text-p mt-2">
-            <v-row align="center" dense>
-              <v-col cols="auto" class="mr-4">
-                <v-btn color="primary" icon="mdi-eye" size="small" variant="text"> </v-btn>
-              </v-col>
-              <v-col>{{ $t('findProjectInstructions.toggleOpacityInstruction') }}.</v-col>
-            </v-row>
-          </div>
+  <v-card-text>
+    <div class="text-p">{{ instructionMessage }}.</div>
+    <div class="text-h6 mt-10">{{ $t('digitizeProjectInstructions.switchMode') }}</div>
+    <div class="text-p mt-2">
+      <v-row align="center" dense>
+        <v-col cols="auto" class="mr-4">
+          <v-btn icon="mdi-draw" color="primary" variant="tonal" rounded="0" />
+        </v-col>
+        <v-col>{{ $t('digitizeProjectInstructions.drawNewFeature') }}</v-col>
+      </v-row>
+      <v-row align="center" v-if="drawType != 'Box'" dense>
+        <v-col cols="auto" class="mr-4">
+          <v-btn icon="mdi-vector-polyline-edit" color="primary" variant="tonal" rounded="0" />
+        </v-col>
+        <v-col>{{ $t('digitizeProjectInstructions.modifyFeature') }}</v-col>
+      </v-row>
+      <v-row align="center" dense>
+        <v-col cols="auto" class="mr-4">
+          <v-btn icon="mdi-delete" color="warning" variant="tonal" rounded="0" />
+        </v-col>
+        <v-col>{{ $t('digitizeProjectInstructions.deleteFeature') }}</v-col>
+      </v-row>
+    </div>
+    <div class="text-h6 mt-10">{{ $t('findProjectInstructions.useButtonsToNavigate') }}</div>
```

Comment:
I'm confused why it says here `findProjectInstructions.useButtonsToNavigate` -> why findProject, as we are in the DigitizeProject. It happens a few more times in this file. Do we need to move these translations to another part as well? e.g. just ProjectInstructions?

Reply:
Agreed, I restructured project instructions translations.